Standards/scs-0112-v1-volume-type-standard.md

```diff
@@ -0,0 +1,220 @@
++--------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| access_project_ids | None                                                                                                                                                         |
+| description        | [encrypted] This volume uses LUKS-encryption                                                                                                                 |
+| id                 | d63307fb-167a-4aa0-9066-66595ea9fb21                                                                                                                         |
+| is_public          | True                                                                                                                                                         |
+| name               | LUKS                                                                                                                                                         |
+| qos_specs_id       | None                                                                                                                                                         |
++--------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------+
+```
+
+### Replication
+
+Replication states, whether or not there are multiple replicas of a volume. Thus answers the question, whether the data could survive a node outage. Unfortunately there are two ways replication can be achieved:
+
+1. In the configuration of a volume type. It then is visible as extra_spec in the properties of a volume type.
+2. Via the used backend. Ceph for example provides automatic replication, that does not need to be specified in the volume type. This is currently not visible for users.
+
+To fulfill this recommentation for now, the deployer needs to state the replication in the description of a volume type in the following way:
+The description needs to begin with `[replicated]`, after that any further description is allowed.
+
+**TODO:**
+We want to find a way to also use the internal extra_spec for replication, when the replication is automatically done by the backend. If this is not possible, we would like to introduce another property, which has to be set by the admin, when setting the volume type. Only after that we will have the possibility to automatically check for a volume type with replication.
+
+#### OPTIONAL addition: Number of Replicas
+
+Additionally to the fact, that a volume type is replicated, it OPTIONALLY can be stated in the description or name, how many times the provisioned volume will be replicated.
```

Comment:
Please add an example.

Reply:
The example is right below, if you think, that is not sufficient i will add another one.